### ISSUE TITLE
Make filters persistent

### ISSUE THREAD

**Issue by nousernameavailableanymore** [nextcloud/deck#2708]
Not sure if this can already be done in any way.
But I usually tend to work with several boards at a time and only want to see specific cards.
Filters in combination with tags work fine for that.

HOWEVER, every time I change to a new board, the previously set filter is reset again (which is super annoying, since all the filter settings need to be manually reapplied again).
Please integrate a little checkbox or similar to make filters persistent between changing / loading boards.

In general I think that Deck is on a good way and has great potential. Keep it up.

**Comment by stefan-niedermann** [comment]
:+1: for persisting the filters **per board**
:-1: for persisting the filters once per application

1. Each board might be different and the filters might not match the use-case of all boards at once (while this might fit for *your personal* use-case, you can't assume that it will fit for all users)
2. Each board can have different labels and users, those are entities which aren't shared by all boards.

**Comment by nousernameavailableanymore** [comment]
Yes, thats why there should be a checkbox in the filter settings for each board (so it can be set for each board individually).

**Comment by nousernameavailableanymore** [comment]
Any news on this one?
